Cargo.toml

```diff
@@ -71,7 +71,10 @@ reedline = { version = "0.25.0", features = ["bashisms", "sqlite"] }
 
 crossterm = "0.27"
 ctrlc = "3.4"
+either = "1.9.0"
```

Comment:
lsp-types does include a `OneOf` enum which might (or might not) do the same job: https://docs.rs/lsp-types/0.94.1/lsp_types/enum.OneOf.html

Reply:
The code uses [`Either::either`](https://docs.rs/either/latest/either/enum.Either.html#method.either) to unify arguments. `OneOf` is only a container and doesn't offer any of these methods.

Additionally, the either crate is already in the dependency tree and I assumed that is okay because it adds no further dependency to Nushell.